### PR TITLE
DEVTOOLS-509: Updating documentation for updating

### PR DESCRIPTION
**Pull request by rjensen26**
Things that have changed
* Brought changes over from `drupal-project-pantheon`
* Adding build guide for each repo that is part of `pantheon` & `f1` hosting.

### CODE REVIEW

File: Dockerfile.pantheon [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+RUN set -ex \
+  && rm -rf node_modules
+
+# Install all dev dependencies for the test image.
+
+FROM gesso AS gesso-dev
+
+#RUN set -ex \
+#  && npm run build-storybook
```
Comment (slucero) [inline Dockerfile.pantheon:62]: Are these supposed to still be commented out?

File: README.f1.md [hunk 2]
```diff
@@ -15,19 +19,10 @@ This is a build guide to help walk you through setting the project up for forumo
 
 Will need to modify the following files on a project bases
```
Comment (slucero) [inline README.f1.md:20]:
```suggestion
Will need to modify the following files on a project basis
```

File: README.f1.md [hunk 3]
```diff
@@ -39,8 +34,8 @@ Will need to modify the following files on a project bases
   * `Dockerfile` and delete any references to `gesso`
 
 Will need to modify the following scripts for the correct `theme location`
-* `.ddev/commands/web/eslint`
-* `.ddev/commands/web/stylelint`
+* `.ddev/commands/gesso/gesso`
+* `.ddev/commands/gesso/gesso`
```
Comment (slucero) [inline README.f1.md:38]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,62 +1,20 @@
-## Development Branch Naming & Workflow
+# Drupal Project Template
 
-- Always create Feature and Bugfix branches from the `main` branch.
-- When needed, create Hotfix branches from `live`.
-- Merge, or submit a Pull Request, from Feature/Bugfix/Hotfix branches to `integration` for Development/Integration environment deployments.
-- Submit a Pull Request from Feature/Bugfix branches to `main` for QA/Staging environment deployments.
-- Submit a Pull Request from Hotfix branches to `live` for Production environment deployments.
-- Submit a Pull Request from `main` to `live` for all regular Production releases.
+This repository holds the scaffolding for newer projects for forumone hosting and pantheon.
```
Comment (slucero) [inline README.md:3]:
```suggestion
This repository holds the scaffolding for newer projects for Forum One hosting and Pantheon.
```

File: README.pantheon.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+  * `.buildkite/pipeline.pantheon.yml` > `.buildkite/pipeline.yml`
+  * `composer.pantheon.json` > `composer.json`
+  * `Dockerfile.pantheon` > `Dockerfile`
+* Please run `ddev config`
+* Need to modify the following variables:
+  * `.buildkite/pipeline.yml` > `[SSH_REPO_FROM_PANTHEON]`
+  * `.buildkite/pipeline.yml` > `[PANTHEON_HOST]`
+  * `.ddev/providers/pantheon.yaml` > `[PROJECT.KEY]`
+* `cp web/sites/default/example.settings.local.php web/sites/default/settings.local.php `
+* Please continue down the readme..
```
Comment (slucero) [inline README.pantheon.md:18]:
```suggestion
* Please continue down the readme.
```

File: README.pantheon.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+
+### Is gesso theme going to be used?
+
+#### If Yes,
+
+Will need to modify the following files on a project bases
```
Comment (slucero) [inline README.pantheon.md:24]:
```suggestion
Will need to modify the following files on a project basis
```

File: README.pantheon.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+#### If No,
+
+* Delete the following files
+  * `.ddev/docker-compose.gesso.yml`
+  * `.ddev/commands/gesso`
+  * `Dockerfile` and delete any references to `gesso`
+
+Will need to modify the following scripts for the correct `theme location`
+* `.ddev/commands/gesso/gesso`
+* `.ddev/commands/gesso/gesso`
```
Comment (slucero) [inline README.pantheon.md:42]:
```suggestion
```

**Review by slucero — APPROVED** [review]

File: .ddev/docker-compose.redis.yml [hunk 8]
```diff
@@ -0,0 +1,20 @@
+version: "3.6"
+services:
+  redis:
+    container_name: ddev-${DDEV_SITENAME}-redis
+    image: redis:5
+    ports:
```
[the remark this answers is not in the document]
Comment (rjensen26) [reply .ddev/docker-compose.redis.yml:6]: So this was the standard approach that the `ddev-contrib` used. https://github.com/drud/ddev-contrib/blob/master/docker-compose-services/redis/docker-compose.redis.yaml

File: Dockerfile.pantheon (hunk 1 above)
Comment (rjensen26) [reply Dockerfile.pantheon:62]: No it is not thanks I have updated it to the correct command.

File: README.f1.md (hunk 3 above)
Comment (rjensen26) [reply README.f1.md:38]: deleted.

**Pull request merged**